.travis.yml

```diff
@@ -4,4 +4,5 @@ before_script:
     - bundle install
 script:
     - xcodebuild -project Aardvark.xcodeproj -scheme Aardvark -sdk iphonesimulator -configuration Debug -PBXBuildsContinueAfterErrors=0 ACTIVE_ARCH_ONLY=0 build test
+    - xcodebuild -project Aardvark.xcodeproj -scheme "Aardvark-iOS" -sdk iphonesimulator -configuration Release -PBXBuildsContinueAfterErrors=0 ACTIVE_ARCH_ONLY=0 build
```

Comment:
Thanks again! Next time I release a library I'll make the main `.h` an umbrella header so folks who come after me don't need to correct my mistake!